payment-request/payment-request-show-method-manual.https.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Test for PaymentRequest.show() method</title>
+<link rel="help" href="https://w3c.github.io/browser-payment-api/#show-method">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+"use strict";
+setup({
+  explicit_done: true,
+  explicit_timeout: true,
+});
+const basicCard = Object.freeze({ supportedMethods: "basic-card" });
+const defaultMethods = Object.freeze([basicCard]);
+const defaultDetails = Object.freeze({
+  total: {
+    label: "Total",
+    amount: {
+      currency: "USD",
+      value: "1.00",
+    },
+  },
+});
+
+test(() => {
+  try {
+    new PaymentRequest(defaultMethods, defaultDetails);
+  } catch (err) {
+    done();
+    throw err;
+  }
+}, "Must be possible to construct a payment request");
+
+promise_test(async t => {
```

Comment:
@domenic, I'm a bit unsure if using `setTimeout` to race these two promises is ok? Basically, a conforming implementation would return `acceptPromise` as rejected with a `SecurityError`. However, in a non-conforming browser, calling `.show()` will show the payment sheet, causing the test to get stuck waiting on `acceptPromise` to resolve. 

In order to avoid the test getting stuck, I call .abort() after 100 milliseconds. Is there a better was of doing this?

Reply:
Generally we let tests timeout in non-conforming browsers. That way, their CI infrastructure can control the timeout period, instead of getting it hard-coded to 100 ms as here.

Reply:
Makes sense. I’ll split this into manual and automated test files. Then the automated can time out.